components/map/MapForPrint.vue

```diff
@@ -1,233 +1,450 @@
 <template>
-    <client-only>
-        <div class="map-container2">
-            <l-map
-                ref="map"
-                :zoom="zoom"
-                :bounds="localBounds"
-                :min-zoom="minZoom"
-                :max-zoom="21"
-                :max-bounds="maxBounds"
-                :max-bounds-viscosity="1"
-                :options="mapOptions"
-                @update:bounds="updateBounds"
+    <v-row no-gutters style="width: 1123px; height: 700px">
+        <v-col cols="8" class="pr-0">
+            <client-only>
+                <l-map
+                    ref="printMap"
+                    id="printMap"
+                    :zoom="zoom"
+                    :options="optionsMap"
+                    :bounds="bounds"
+                >
+                    <l-tile-layer
+                        url="http://{s}.tile.osm.org/{z}/{x}/{y}.png"
+                        :attribution="attribution"
+                    ></l-tile-layer>
+
+                    <l-control-scale v-if="valueScale" position="bottomleft" />
+                    <l-control v-if="valueNorthArrow" position="bottomleft">
+                        <img
+                            src="../../assets/north-arrow.png"
+                            alt="northarrow"
+                            class="north-arrow"
+                        />
+                    </l-control>
+                    <PriorityLayers :map="map" />
+                    <MonitoringLayers :map="map" />
+                    <SupportLayers />
+                    <AlertLayers :map="map" />
+                </l-map>
+            </client-only>
+        </v-col>
+        <v-col cols="4" class="pl-1">
+            <div
+                style="
+                    border-right: 0.5px solid gray;
+                    border-top: 0.5px solid gray;
+                    border-bottom: 0.5px solid gray;
+                    height: 100%;
+                "
             >
-                <l-control position="topleft">
-                    <div class="pa-1 map-action-buttons">
-                        <div
-                            v-if="user.settings.map_zoom_buttons_visible"
-                        ></div>
-
-                        <div class="div-spacer" />
+                <div class="d-flex justify-center align-center ma-2">
+                    <div style="width: 20%">
+                        <v-img contain class="mr-4" src="/img/funai.svg" />
                     </div>
-                </l-control>
-
-                <l-control
-                    class="leaflet-coordinates-control"
-                    position="bottomleft"
-                >
-                    <div v-if="user.settings.map_pointer_coordinates_visible">
-                        {{ cursorCoordinates.lat }},
-                        {{ cursorCoordinates.lng }}
+                    <div style="width: 25%">
```

Comment:
inline style? Why dont you use vue style?

Reply:
Used inline because I don't see the need to create several classes just to define the width of the container. The lib (domtoimage) generates the image according to the size of the elements so it was also necessary to define a fixed width.